### PR TITLE
Build wanda with premium checks, if available

### PR DESCRIPTION
**Pull request by nelsonkopliku**
Allows extending the Checks Catalog with premium Checks when available.

### CODE REVIEW

File: packaging/suse/Dockerfile [hunk 1]
```diff
@@ -5,7 +5,8 @@
 #!UseOBSRepositories
 
 FROM bci/rust:1.66 AS release
-ADD wanda.tar.gz /build/
+ADD wanda.tar.gz premium-checks.tar.gz* /build/
```
Comment (arbulu89) [inline packaging/suse/Dockerfile:8]: It wouldn't be better:
```
ADD wanda.tar.gz /build/
ADD premium-checks.tar.gz* /build/wanda/
```
As the `ADD` command extract the tar file in the destination folder.
Didn't test it though

Comment (nelsonkopliku) [reply packaging/suse/Dockerfile:8]: Tested and doesn't work when `premium-checks.tar.gz` is not there. Docker wants at least the first item for an `ADD` to actually be there.

Comment (arbulu89) [reply packaging/suse/Dockerfile:8]: ok!
Thank you for trying out

**Review by arbulu89 — APPROVED** [review]

**Review by dottorblaster — APPROVED** [review]

**Review by rtorrero — APPROVED** [review]
LGTM

**Review by EMaksy — COMMENTED** [review]
LGTM :D

**Pull request merged**